PerformanceTesting/ecl/02eb_smallsorts2.ecl

```diff
@@ -15,17 +14,121 @@
 //version algo='parmergesort',numrows=6000
 //version algo='parmergesort',numrows=10000
 
+//xversion algo='parmergesort',numrows=12000
```

Comment:
Why these "//xversion ..." ineffective tags are added?

Reply:
They are commented out as different option that I was using to see how the sorts scaled with the number of records.  The number of tests is excessive for a general run through.